tests/common/schedulers/test_slurm_commands.py

```diff
@@ -194,23 +194,23 @@ def test_is_static_node(nodename, expected_is_static):
             False,
         ),
         (
-            "NodeName=multiple-dy-c5xlarge-4\n"
-            "NodeAddr=multiple-dy-c5xlarge-4\n"
-            "NodeHostName=multiple-dy-c5xlarge-4\n"
+            "NodeName=multiple-dy-c5xlarge-3\n"
```

Comment:
Why did you change the name from `multiple-dy-c5xlarge-4` to `multiple-dy-c5xlarge-3`?